### PR TITLE
Updated README.

### PR DESCRIPTION
**Pull request by RaghuvirShirodkar**
[README] Modified to reflect possible errors encountered during installation and their solutions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,3 +138,109 @@ In a new terminal you can open the Gazebo GUI to see the environment and the rob
...
+### Errors related to uninstalled packages
+There are certain packages needed as prerequisite. 
+
+_Solution_: In case you do not have those, please google the relevant package, clone the git repo to your local machine, then build it. Following packages are generally found to be missing.
+
+#### [brics_actuator]
+[brics_actuator]: http://wiki.ros.org/brics_actuator
+Clone the git repo to your machine:
+
+     git clone git@github.com:wnowak/brics_actuator.git
```
Comment (DharminB) [inline README.md:173]: There is a `ros-kinetic-brics-actuator` package in `apt`. Why shouldn't one just install using `sudo apt install ros-kinetic-brics-actuator` instead?

File: README.md [hunk 2]
```diff
@@ -138,3 +138,109 @@ In a new terminal you can open the Gazebo GUI to see the environment and the rob
...
+Then continue building remaining packages
+
+     roscd && cd ../
+     catkin build
+
+#### [geometry2]
+[geometry2]: http://wiki.ros.org/geometry2
+Clone the git repo to your machine:
+
+     git clone git@github.com:ros/geometry2.git
```
Comment (DharminB) [inline README.md:210]: Same as above. `sudo apt install ros-kinetic-geometry2`

File: README.md [hunk 3]
```diff
@@ -138,3 +138,109 @@ In a new terminal you can open the Gazebo GUI to see the environment and the rob
...
+Then continue building remaining packages
+
+     roscd && cd ../
+     catkin build
+
+#### [laser_filters]
+[laser_filters]: http://wiki.ros.org/laser_filters
+Clone the git repo to your machine:
+
+     git clone git@github.com:ros-perception/laser_filters.git
```
Comment (DharminB) [inline README.md:226]: Same as above. `sudo apt install ros-kinetic-laser-filters`

File: README.md (hunk 1 above)
Comment (deebuls) [reply README.md:173]: @RaghuvirShirodkar  ?

**Comment by deebuls** [comment]
@DharminB  I think this is pretty old .. I will close this pull request . Since we have moved to docker based one.

**Comment by DharminB** [comment]
> @DharminB I think this is pretty old .. I will close this pull request . Since we have moved to docker based one.

It's @RaghuvirShirodkar 's PR. I just reviewed it. I don't have any problem with you closing it. :blush:

**Pull request closed without merging**